src/app.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+import router from "./views/pages/router.js";
+import Error404 from "./views/pages/error404.js";
+
+const renderPage = async () => {
+  let view = router.match();
+  const root = document.getElementById('root');
+  if (view != null) {
```

Comment:
!==

Reply:
This was a work in progress. Shouldn't have been in this PR. belongs in the SPA PR, i.e. pull/3.